pkg/client/http/http.go

```diff
@@ -112,10 +111,13 @@ func (dc *Client) Call(req *client.Request) (resp interface{}, err error) {
 	params, _ := transformedParams.(*requestParams)
 	request.Body = params.Body
 	request.Header = params.Header
-	// url query add.
-	urlStr = strings.TrimRight(u.String(), "/") + "?" + params.Query.Encode()
 
-	newReq, err := http.NewRequest(req.IngressRequest.Method, urlStr, params.Body)
+	targetURL, err := dc.parseURL(req, *params)
+	if err != nil {
+		return nil, err
+	}
+
+	newReq, err := http.NewRequest(req.IngressRequest.Method, targetURL, params.Body)
 
 	httpClient := &http.Client{Timeout: 5 * time.Second}
```

Comment:
a little tip: move this timeout option to dc

Reply:
I think It should be a configuration read from the config file or get the default value.